dht/murmur3_partitioner.cc

```diff
@@ -78,13 +70,7 @@ murmur3_partitioner::get_token(bytes_view key) const {
 
 token
 murmur3_partitioner::get_token(uint64_t value) const {
```

Comment:
This commit does more than just changing `_data` to `int64_t`, and the commit message doesn't mention anything about that, forcing reviewers to guess...

This commit changes `token` to store the data in host byte order, where previously it stored data in network byte order.

Reply:
I added commit message and extracted the normalization logic to a function as before.